src/runtime/audience-action-flow.js

```diff
@@ -140,10 +140,32 @@ export class AudienceActionFlow {
       }
 
       this.activityIframeView_ = activityIframeView;
-      return this.dialogManager_.openView(activityIframeView);
+      return this.clientConfigManager_
+        .getClientConfig()
+        .then((clientConfig) => {
+          if (!this.activityIframeView_) {
+            return;
+          }
+          return this.dialogManager_.openView(
+            this.activityIframeView_,
+            /* hidden */ false,
+            this.getDialogConfig_(clientConfig)
+          );
+        });
     });
   }
 
+  /**
+   *
+   * @param {!../model/client-config.ClientConfig} clientConfig
+   * @return {!../components/dialog.DialogConfig}
+   */
+  getDialogConfig_(clientConfig) {
+    return clientConfig.useUpdatedOfferFlows
```

Comment:
I don't have much context around what's the original/updated offer flows.
Would you explain when we want to allow body scrolling? If this is not necessary, the change can be as simple as 
```
this.dialogManager_.openView(
  activityIframeView, 
  /* hidden */ false, 
  /* dialogConfig */ {
    shouldDisableBodyScrolling: true
  });
```